docs/services/vision/classification.md

```diff
@@ -88,7 +88,26 @@ Proceed to [test your classifier](#test-your-classifier).
 
 ## Test your classifier
 
-You can test your classifier with [live camera footage](#live-camera-footage) or [existing images](#existing-images).
+You can test your classifier with [existing images in the Viam app](#existing-images-in-the-cloud), [live camera footage](#live-camera-footage), or [existing images on a computer](#existing-images-on-your-machine).
+
+### Existing images in the cloud
+
+{{< alert title="Note" color="note" >}}
+
+Your classifier must have been trained or uploaded after the September 19, 2023.
```

Comment:
```suggestion
The feature is only available for classifiers that were uploaded after September 19, 2023.
```

Unless the timestamp of training is also considered in the availability of this feature.